Dockerfile.development

```diff
@@ -11,9 +10,12 @@ WORKDIR /gossamer
 
 COPY . .
 
-RUN go get ./...
-RUN go install -trimpath github.com/ChainSafe/gossamer/cmd/gossamer
+ARG GO_BUILD_FLAGS
+RUN go build \
+    -trimpath \
+    -o /gossamer/bin/gossamer \
+    ${GO_BUILD_FLAGS} \
+    ./cmd/gossamer
 
-RUN ["sh", "-c", "gossamer init --chain=${chain} --base-path=${basepath}/${chain}"]
-ENTRYPOINT ["sh", "-c", "gossamer --chain=${chain} --base-path=${basepath}/${chain} --rpc-external=true --unsafe-rpc=true --prometheus-external --prometheus-port=9876 --pprof.enabled --pprof.listening-address=\":6060\""]
```

Comment:
The `entrypoint` now directly runs the binary, so any flags or arguments must be passed explicitly via the `command` field in the task definition for the ECS case, and ansible task role.

Reply:
I also thought that this was what we discussed on our last call: Make the `entrypoint` the same as in the "main" [Dockerfile](https://github.com/ChainSafe/gossamer/blob/2eff00475ac1234ac1701f596808f93b3bf0bd46/Dockerfile#L44). If that causes you extra work, then we shouldn't do it.

How should this line look ideally from your perspective?

Reply:
This should be fine, I've prepared some changes that should make this work. You can merge these changes safely.